sound/soc/intel/boards/Kconfig

```diff
@@ -536,6 +536,30 @@ config SND_SOC_INTEL_TGL_RT711_RT1308_MACH
 
 endif ## SND_SOC_SOF_TIGERLAKE && SND_SOC_SOF_INTEL_SOUNDWIRE
 
+if SND_SOC_SOF_ELKHARTLAKE
+
+config SND_SOC_INTEL_EHL_RT5660_MACH
+	tristate "EHL with RT5660 in I2S mode"
+	depends on I2C && ACPI
+	depends on MFD_INTEL_LPSS || COMPILE_TEST
+	select SND_SOC_RT5660
```

Comment:
where are all the HDMI dependencies?

Reply:
@libinyang Can you comment on this? Based on code, it's completely optional now (although some fixes are needed to really make it optional).

Reply:
@plbossart  We discussed the options of making HDMI optional or having a mandatory kconfig dependency. Now we went with simple depends on. This should be ok as you can disable HDMI at runtime via codec_mask.